presidio-analyzer/tests/test_pl_pesel_recognizer.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from tests import assert_result
+from presidio_analyzer.predefined_recognizers import PlPeselRecognizer
+
+
+@pytest.fixture(scope="module")
+def recognizer():
+    return PlPeselRecognizer()
+
+
+@pytest.fixture(scope="module")
+def entities():
+    return ["PL_PESEL"]
+
+
+@pytest.mark.parametrize(
+    "text, expected_len, expected_positions",
+    [
+        # fmt: off
+        # valid PESEL scores
+        ("11111111114", 1, ((0, 11),),),
+        # invalid PESEL scores
```

Comment:
Please add cases where the pesel number is in the middle of the sentence (e.g. `my pesel is 11111111114`)